hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/rate/maxrate/MaxRate.java

```diff
@@ -0,0 +1,42 @@
+package pl.allegro.tech.hermes.consumers.consumer.rate.maxrate;
+
+import java.beans.ConstructorProperties;
+import java.util.Objects;
+
+final class MaxRate {
+
+    private final double maxRate;
+
+    @ConstructorProperties({"maxRate"})
+    MaxRate(double maxRate) {
+        this.maxRate = maxRate;
+    }
+
+    public double getMaxRate() {
+        return maxRate;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
It's used in test for instance. I prefer to have these methods here, as they're not complex (discarding YAGNI makes sense IMO here)